.github/workflows/deploy.yml

```diff
@@ -34,37 +34,31 @@ jobs:
       - name: Run unit tests
         run: npm run test:ci
 
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
+      - uses: actions/upload-artifact@v2
+        if: github.ref == 'refs/heads/master'
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: eu-west-2
-
-      - name: Log in to Amazon ECR
-        id: login-ecr
-        uses: aws-actions/amazon-ecr-login@v1
-
-      - name: Docker Build, tag and push image to Amazon ECR
-        env:
-          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-          ECR_REPOSITORY: tis-revalidation-v2
-          IMAGE_TAG: ${{ github.sha }}
-        run: |
-          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
-          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
-          docker tag $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG $ECR_REGISTRY/$ECR_REPOSITORY:latest
-          docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
+          name: revalidation-artifact:${{ github.sha }}
+          path: dist/revalidation/ # or dist
 
   deploy:
     name: Deploy
     needs: build
     runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
 
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - uses: actions/download-artifact@v2
+        with:
+          name: revalidation-artifact:${{ github.sha }}
+          path: dist/revalidation/
+
+      - name: Display structure of downloaded files
+        run: ls -R
+        working-directory: dist/revalidation/
```

Comment:
Happy to leave it, but I'm not sure how useful listing everything will be?

Reply:
removed